tests/unit/test_vault.py

```diff
@@ -141,6 +141,25 @@ def test_given_audit_device_is_not_yet_enabled_when_enable_audit_device_then_dev
             device_type="file", options={"file_path": "stdout"}
         )
 
+    @patch("hvac.api.system_backend.Raft.take_raft_snapshot")
+    def test_given_raft_backend_when_create_snapshot_then_take_raft_snapshot_is_called(
+        self,
+        patch_take_raft_snapshot,
+    ):
+        vault = Vault(url="http://whatever-url", ca_cert_path="whatever path")
+        vault.create_snapshot()
+        patch_take_raft_snapshot.assert_called_once()
```

Comment:
I'm not sure there's much value in tests like these

Reply:
Removed